src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -406,11 +411,24 @@ public async Task UnlinkUserAsync(UnlinkUserRequest request)
         }
 
         /// <summary>
-        /// Disposes of any owned disposable resources such as the ApiConnection.
+        /// Dispose of any managed resources such as the <see cref="IApiConnection"/>.
+        /// </summary>
+        /// <param name="disposing">Whether to dispose of managed resources right now.</param>
+        protected virtual void Dispose(bool disposing)
```

Comment:
I'm confused why this is necessary. Why would you call this with `disposing` set to false?

Reply:
It's the standard .NET pattern for disposing objects that can be inherited from.  VS suggests it strongly from the IDE.